app.py

```diff
@@ -544,8 +612,9 @@ def annotate(sentno):
 			worker.domorph(item.tree)
 			tree = writediscbrackettree(item.tree, item.sent)
 		else: 
-			tree = annotation
-			_, senttok = brackettree( "(ROOT" + cgel.parse(annotation)[0].ptb(punct = True) + ")" )
+			treeobj = ActivedopTree.from_str(annotation)
+			senttok = treeobj.senttok
```

Comment:
is this being used?

Reply:
Good point -- no, there was no reason to define the `senttok` variable here. Deleted in https://github.com/nschneid/activedop/pull/73/commits/08310ce969e66f05a53402bf2dab0b554b3e83d1